Dockerfile

```diff
@@ -29,4 +29,4 @@ COPY --chown=node:node ./config ./config
 
 USER node
 EXPOSE 8080
-CMD ["dumb-init", "node", "--max_old_space_size=512", "--require", "./common/tracing.js", "./index.js"]
+CMD ["dumb-init", "node", "--max_old_space_size=4096", "--require", "./common/tracing.js", "./index.js"]
```

Comment:
Please recheck with the 512 and revert if needed

Reply:
changed